### PR TITLE
20w28a worldgen mappings

### PR DESCRIPTION
**Pull request by Juuxel**
~~This has some conflicts with #1562 that I'll fix later. Also somewhat overlaps with #1568.~~

No conflicts anymore :tiny_potato:

### CODE REVIEW

**Review by liach — APPROVED** [review]

**Review by shartte — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping [hunk 1]
```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5428 net/minecraft/world/gen/UniformIntDistribution
```
Comment (liach) [inline mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:1]: Can't agree with this name. My first instinct on seeing "base" and spread given this name would be that base is the center and spread is the radius. But it is not.

Comment (liach) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:1]: This is more like a "base" and a "bonus" combination

Comment (shartte) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:1]: It's still a uniform integer distribution though.
I think the name is fine, even though it's Constrained/Clamped

Comment (liach) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:1]: Can we get like a shorter name for this? This long name looks weird when I have to use it

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	FIELD field_25809 CODEC Lcom/mojang/serialization/Codec;
+	FIELD field_25810 base I
+	FIELD field_25811 spread I
+	METHOD <init> (II)V
+		ARG 1 base
+		ARG 2 spread
+	METHOD method_30314 of (I)Lnet/minecraft/class_5428;
```
Comment (liach) [inline mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:8]: if you keep this class name, i'd suggest `fixed` which may be a bit more descriptive

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+		ARG 0 base
+	METHOD method_30315 of (II)Lnet/minecraft/class_5428;
+		ARG 0 base
+		ARG 1 spread
+	METHOD method_30316 createCodec (III)Lcom/mojang/serialization/Codec;
```
Comment (liach) [inline mappings/net/minecraft/world/gen/UniformIntDistribution.mapping]: `createValidatedCodec`?

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping (hunk 2 above)
Comment (Juuxel) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:8]: Maybe `constant`?

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping (hunk 3 above)
Comment (Juuxel) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping]: Sure, I'll change it to that.

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:8]: just keep it `of` then

File: mappings/net/minecraft/world/gen/feature/CountFeatureConfig.mapping [hunk 4]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_2984 net/minecraft/world/gen/feature/CountFeatureConfig
+	FIELD field_13385 count Lnet/minecraft/class_5428;
+	FIELD field_24878 CODEC Lcom/mojang/serialization/Codec;
+	METHOD <init> (I)V
+		ARG 1 count
+	METHOD <init> (Lnet/minecraft/class_5428;)V
+		ARG 1 count
+	METHOD method_30396 getCount ()Lnet/minecraft/class_5428;
```
Comment (YanisBft) [inline mappings/net/minecraft/world/gen/feature/CountFeatureConfig.mapping]: This class has already be mapped and is named `CountConfig`

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping (hunk 1 above)
Comment (Juuxel) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:1]: Any suggestions?

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping (hunk 2 above)
Comment (Juuxel) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping:8]: So, should I rename it or not?

File: mappings/net/minecraft/world/gen/feature/CountFeatureConfig.mapping (hunk 4 above)
Comment (Juuxel) [reply mappings/net/minecraft/world/gen/feature/CountFeatureConfig.mapping]: Yeah, I haven't yet merged this with master properly.

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/world/gen/UniformIntDistribution.mapping (hunk 3 above)
Comment (Juuxel) [reply mappings/net/minecraft/world/gen/UniformIntDistribution.mapping]: Changed to `createValidatedCodec`.

**Review by YanisBft — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**